src/SerilogWeb.Classic/Classic/ApplicationLifecycleModule.cs

```diff
@@ -36,6 +37,7 @@ public class ApplicationLifecycleModule : IHttpModule
         static volatile IEnumerable<string> _filteredKeywords = new[] { "password" };
         static volatile LogEventLevel _requestLoggingLevel = LogEventLevel.Information;
         static volatile LogEventLevel _formDataLoggingLevel = LogEventLevel.Debug;
```

Comment:
I think the number of memory barriers these are generating is getting a bit out of hand, perhaps we drop all the `volatile`s and just accept a very rare chance of "delayed action" when configuring these (almost impossible in ASP.NET apps).